### PR TITLE
Rename "develop" to "advanced"

### PR DESCRIPTION
**Pull request by roberthein**

### CODE REVIEW

File: Toshi/Controllers/Settings/Settings.storyboard [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12121" systemVersion="16F73" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="h2G-MK-F2K">
```
Comment (yuliaveres) [inline Toshi/Controllers/Settings/Settings.storyboard]: what is the purpose of this file changes ?

Comment (roberthein) [inline Toshi/Controllers/Settings/Settings.storyboard]: Maybe it was last opened with Xcode 8.3.2?

Comment (roberthein) [inline Toshi/Controllers/Settings/Settings.storyboard]: @yuliaveres What are you running?

Comment (yuliaveres) [inline Toshi/Controllers/Settings/Settings.storyboard]: nope, xcode applies those weird changes to any IB file you open. and I prefer discard those if you didn't add anything to the file 🙏

File: Toshi/Controllers/Settings/Settings.storyboard [hunk 2]
```diff
@@ -183,10 +183,10 @@
...
-                                            <rect key="frame" x="0.0" y="0.0" width="342" height="43.5"/>
+                                            <rect key="frame" x="0.0" y="0.0" width="342" height="44"/>
                                             <autoresizingMask key="autoresizingMask"/>
                                             <subviews>
-                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Develop" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="cRs-tP-aau">
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Advanced" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="cRs-tP-aau">
```
Comment (yuliaveres) [inline Toshi/Controllers/Settings/Settings.storyboard]: u changed the text, so that is needed

**Pull request merged**